src/core/Song.cpp

```diff
@@ -1438,12 +1476,10 @@ void Song::setProjectFileName(QString const & projectFileName)
 	{
 		m_fileName = projectFileName;
 		emit projectFileNameChanged();
-	}
+    }
```

Comment:
Nope. We use tabs! Please change every instance of indents with spaces to tabs instead.